src/main/java/com/foundation/trello/util/NamesGenerator.java

```diff
@@ -0,0 +1,70 @@
+package com.foundation.trello.util;
+
+import com.github.javafaker.Faker;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * This class generates random names with their respective suffixes and prefixes.
+ *
+ * @author Maday Alcala.
+ * @version 0.0.1
+ */
+public final class NamesGenerator {
+    private static String newName;
+    private static String suffix;
+    private static String prefix;
+    private static Date actual;
+    private static Faker faker;
+
+    /**
+     * This is the constructor method that initializes the variables.
+     */
+    private NamesGenerator() {
+    }
+
+    /**
+     * This method is in charge of joining a suffix with a random name and its prefix
+     * in the following format: <prefix>_<name>_<suffix>.
+     *
+     * @return 'newName' that represent a name with its respective suffix and prefix.
+     */
+    public static String newName() {
+        newName = prefix() + " " + aleatoryName() + " " + suffix();
+        return newName;
+    }
+
+    /**
+     * This class returns the string "AT10" as prefix.
+     *
+     * @return a prefix in string type.
+     */
+    public static String prefix() {
```

Comment:
change this method and use a constant please: E.g.  public static final String PREFIX= "AT-10";

Reply:
Solved.